eng/testing/RunnerTemplate.sh

```diff
@@ -252,18 +204,47 @@ if [[ "$(uname -s)" == "Linux" && $test_exitcode -ne 0 ]]; then
   if [[ "$core_name_uses_pid" == "1" ]]; then
     # We don't know what the PID of the process was, so let's look at all core
     # files whose name matches core.NUMBER
-    echo Looking for files matching core.* ...
-    for f in core.*; do
-      [[ $f =~ core.[0-9]+ ]] && print_info_from_core_file "$f" "dotnet" && copy_core_file_to_temp_location "$f" && rm "$f"
+    echo "Looking for files matching core.*"
+    for f in $(find . -name "core.*"); do
+      [[ $f =~ core.[0-9]+ ]] && move_core_file_to_temp_location "$f"
```

Comment:
Why do you need this?  I thought above with `DOTNET_DbgMiniDumpName` you configure the dump to go into `HELIX_DUMP_FOLDER` why would dump files need to be moved from the working directory?

Reply:
This is trying to fixup system dumps - up to y'all if you want to keep those around.

Reply:
I think it doesn't hurt to move them to the temp location. If later we notice they add too much noise and not much helpful info, we can remove those 3 lines.